rest_framework_tracking/admin.py

```diff
@@ -28,9 +28,8 @@ class APIRequestLogAdmin(admin.ModelAdmin):
     list_filter = ("view_method", "status_code")
     search_fields = (
         "path",
-        "user__email",
+        "user",
     )
-    raw_id_fields = ("user",)
```

Comment:
What does this do

Reply:
this caused migration error


ERRORS:
<class 'rest_framework_tracking.admin.APIRequestLogAdmin'>: (admin.E003) The value of 'raw_id_fields[0]' must be a foreign key or a many-to-many field.
<class 'rest_framework_tracking.admin.APIRequestLogAdmin'>: (admin.E003) The value of 'raw_id_fields[0]' must be a foreign key or a many-to-many field.